keyboards/cannonkeys/tmov2/tmov2.h

```diff
@@ -3,13 +3,13 @@
 #include "quantum.h"
 
 #define LAYOUT_default( \
-	K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, \
-	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113,       \
-	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213,       \
-	K300,       K302, K303,       K305,       K307,       K309, K310                            \
+	K000, K001,       K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, \
+	K100, K101,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113,       \
+	K200, K201, K215, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213,       \
+	K300,             K302, K303,       K305,       K307,       K309, K310                            \
 ) { \
-	{ K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014 }, \
-	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  KC_NO}, \
-	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  KC_NO}, \
-	{ K300,  KC_NO,  K302,  K303,  KC_NO,  K305,  KC_NO,  K307,  KC_NO,  K309,  K310,  KC_NO,  KC_NO,  KC_NO} \
+	{ K000,  K001,  KC_NO, K002, K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014 }, \
+	{ K100,  K101,  KC_NO, K102, K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  KC_NO}, \
+	{ K200,  K201,  K215,  K202,   K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  KC_NO}, \
+	{ K300,  KC_NO, KC_NO, K302, K303,  KC_NO, K305,  KC_NO, K307,  KC_NO, K309,  K310,  KC_NO, KC_NO, KC_NO, KC_NO} \
 }
```

Comment:
For Lines 5-15:

```c
#define LAYOUT_default( \
    K000, K001,       K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, \
    K100, K101,       K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114,       \
    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214,       \
    K300,             K303, K304,       K306,       K308,       K310, K311                          \
) { \
    { K000,  K001,  KC_NO,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014,  K015  }, \
    { K100,  K101,  KC_NO,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  K114,  KC_NO }, \
    { K200,  K201,  K202,   K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  K214,  KC_NO }, \
    { K300,  KC_NO, KC_NO,  K303,  K304,  KC_NO, K306,  KC_NO, K308,  KC_NO, K310,  K311,  KC_NO, KC_NO, KC_NO, KC_NO }  \
}
```

Renumbers the matrix positions so the row and column numbers are accurate again. Zero logic change.